pkg/controller/certificatemanager/certificatemanager.go

```diff
@@ -365,3 +435,53 @@ func (cm *certificateManager) CreateTrustedBundle(certificates ...certificateman
 func (cm *certificateManager) CreateTrustedBundleWithSystemRootCertificates(certificates ...certificatemanagement.CertificateInterface) (certificatemanagement.TrustedBundle, error) {
 	return certificatemanagement.CreateTrustedBundleWithSystemRootCertificates(append([]certificatemanagement.CertificateInterface{cm.keyPair}, certificates...)...)
 }
+
+func (cm *certificateManager) LoadTrustedBundle(ctx context.Context, client client.Client, ns string) (certificatemanagement.TrustedBundleRO, error) {
+	obj := &corev1.ConfigMap{}
+	k := types.NamespacedName{Name: certificatemanagement.TrustedCertConfigMapName, Namespace: ns}
+	if err := client.Get(ctx, k, obj); err != nil {
+		return nil, err
+	}
+	a := newReadOnlyTrustedBundle(cm, len(obj.Data[certificatemanagement.RHELRootCertificateBundleName]) > 0)
+	for key, val := range obj.Annotations {
```

Comment:
The reason this structure was introduced was because it's impossible (or really hard, at least) to re-build a `TrustedBundle` interface from the ConfigMap loaded into the cluster. The fact that it's RO isn't the goal of this interface, just a (useful) consequence of needing to be able to load metadata from an existing bundle configmap. 

The annotations on the ConfigMap are a new change I added in this PR in order to enable this particular struct - they should be exactly the same as what would otherwise be created, assuming I implemented this correctly.